### PR TITLE
Update Russian Localization

### PR DESCRIPTION
**Pull request by denis-ershov**

### CODE REVIEW

File: src/GwentTracker/locale/cards.ru.po [hunk 1]
```diff
@@ -385,51 +385,51 @@ msgstr "Мелкий, быстрый и зловредный."
...
+msgstr "Труп Хаммонда"
 
 #: src/GwentTracker/data/monsters.yml:323 -> Card.area
 msgid "Trottheim"
 msgstr "Деревня Троттхейм"
 
 #: src/GwentTracker/data/monsters.yml:353 -> Card.name
 msgid "Vampire: Ekimmara"
-msgstr "Вампиры: экимма"
+msgstr "Вампиры: Экимма"
```
Comment (rfvgyhn) [inline src/GwentTracker/locale/cards.ru.po]: This one doesn't match the in-game name (uppercase vs lowercase _Э_). This change will be overwritten by the script that pulls translations directly from the game's w3strings files.

File: src/GwentTracker/locale/cards.ru.po [hunk 2]
```diff
@@ -385,51 +385,51 @@ msgstr "Мелкий, быстрый и зловредный."
...
 
 #: src/GwentTracker/data/monsters.yml:354 -> Card.flavor
 msgid "Who would think overgrown bats would have a taste for gaudy jewelry?"
 msgstr "Экимма - она вроде как нетопырь... Только размером с человека."
 
 #: src/GwentTracker/data/monsters.yml:365 -> Card.name
 msgid "Vampire: Fleder"
-msgstr "Вампиры: фледер"
+msgstr "Вампиры: Фледер"
```
Comment (rfvgyhn) [inline src/GwentTracker/locale/cards.ru.po]: Same as _Vampire: Ekimmara_

File: src/GwentTracker/locale/cards.ru.po [hunk 3]
```diff
@@ -385,51 +385,51 @@ msgstr "Мелкий, быстрый и зловредный."
...
 
 #: src/GwentTracker/data/monsters.yml:366 -> Card.flavor
 msgid "Higher vampires embrace their victims. Fleders rip them to shreds."
 msgstr "Лысая башка, шипастые уши, шкура в бородавках. Одно слово - красавчик."
 
 #: src/GwentTracker/data/monsters.yml:377 -> Card.name
 msgid "Vampire: Garkain"
-msgstr "Вампиры: гаркаин"
+msgstr "Вампиры: Гаркаин"
```
Comment (rfvgyhn) [inline src/GwentTracker/locale/cards.ru.po]: Same as Vampire: Ekimmara

File: src/GwentTracker/locale/cards.ru.po [hunk 4]
```diff
@@ -385,51 +385,51 @@ msgstr "Мелкий, быстрый и зловредный."
...
 
 #: src/GwentTracker/data/monsters.yml:378 -> Card.flavor
 msgid "Blood-drinkers and corpse-eaters so foul their very ugliness paralyses foes."
 msgstr "Славно, что вы сразу же нашпиговалися, потому как теперича я вас жрать стану."
 
 #: src/GwentTracker/data/monsters.yml:389 -> Card.name
 msgid "Vampire: Bruxa"
-msgstr "Вампиры: брукса"
+msgstr "Вампиры: Брукса"
```
Comment (rfvgyhn) [inline src/GwentTracker/locale/cards.ru.po]: Same as Vampire: Ekimmara

File: src/GwentTracker/locale/cards.ru.po [hunk 5]
```diff
@@ -385,51 +385,51 @@ msgstr "Мелкий, быстрый и зловредный."
...
 
 #: src/GwentTracker/data/monsters.yml:399 -> Card.notes
 msgid "Win a match at the masquerade ball at Vegelbud Estate"
-msgstr ""
+msgstr "Выиграйте матч на бал-маскараде в поместье Вегельбуд"
 
 #: src/GwentTracker/data/monsters.yml:403 -> Card.name
 msgid "Vampire: Katakan"
-msgstr "Вампиры: катакан"
+msgstr "Вампиры: Катакан"
```
Comment (rfvgyhn) [inline src/GwentTracker/locale/cards.ru.po]: Same as Vampire: Ekimmara

File: src/GwentTracker/locale/cards.ru.po [hunk 6]
```diff
@@ -489,7 +489,7 @@ msgstr "Эредин Бреакк Глас Король Aen Elle"
 
 #: src/GwentTracker/data/monsters.yml:489 -> Card.flavor
 msgid "I've long awaited this..."
-msgstr "Va faill, luned."
+msgstr "Я давно этого ждал..."
```
Comment (rfvgyhn) [inline src/GwentTracker/locale/cards.ru.po]: Same as Vampire: Ekimmara

File: src/GwentTracker/locale/cards.ru.po [hunk 7]
```diff
@@ -561,15 +561,15 @@ msgstr "Вроницы"
 
 #: src/GwentTracker/data/neutral.yml:21 -> Card.area
 msgid "Seven Cats Inn"
-msgstr "Корчма Семь котов"
+msgstr "Корчма Семь Котов"
```
Comment (rfvgyhn) [inline src/GwentTracker/locale/cards.ru.po]: Same as Vampire: Ekimmara

File: src/GwentTracker/locale/cards.ru.po [hunk 8]
```diff
@@ -561,15 +561,15 @@ msgstr "Вроницы"
...
 
 #: src/GwentTracker/data/neutral.yml:22 -> Card.territory
 msgid "Gustfields"
-msgstr ""
+msgstr "Золотые поля"
 
 #: src/GwentTracker/data/neutral.yml:27 -> Card.name
 msgid "Commander's Horn"
-msgstr "Командирский рог"
+msgstr "Командирский Рог"
```
Comment (rfvgyhn) [inline src/GwentTracker/locale/cards.ru.po]: Same as Vampire: Ekimmara

**Review by rfvgyhn — CHANGES_REQUESTED** [review]
Looks great. Just have a few comments.

File: src/GwentTracker/locale/cards.ru.po (hunk 1 above)
Comment (W1ns-Smile) [reply src/GwentTracker/locale/cards.ru.po]: In Russian localization, Ekimmara translates as Ekimma
https://vedmak.fandom.com/wiki/Экимма

File: src/GwentTracker/locale/cards.ru.po (hunk 8 above)
Comment (W1ns-Smile) [reply src/GwentTracker/locale/cards.ru.po]: [Image link](https://vignette.wikia.nocookie.net/vedmak/images/e/eb/33%D0%BA%D0%BE%D0%BC._%D1%80%D0%BE%D0%B3.png/revision/latest?cb=20150623144441)

File: src/GwentTracker/locale/cards.ru.po (hunk 7 above)
Comment (W1ns-Smile) [reply src/GwentTracker/locale/cards.ru.po]: https://vedmak.fandom.com/wiki/Корчма_«Семь_котов»

File: src/GwentTracker/locale/cards.ru.po (hunk 5 above)
Comment (W1ns-Smile) [reply src/GwentTracker/locale/cards.ru.po]: https://vedmak.fandom.com/wiki/Катакан

File: src/GwentTracker/locale/cards.ru.po (hunk 4 above)
Comment (W1ns-Smile) [reply src/GwentTracker/locale/cards.ru.po]: https://vedmak.fandom.com/wiki/Брукса

File: src/GwentTracker/locale/cards.ru.po (hunk 3 above)
Comment (W1ns-Smile) [reply src/GwentTracker/locale/cards.ru.po]: https://vedmak.fandom.com/wiki/Гаркаин

File: src/GwentTracker/locale/cards.ru.po (hunk 2 above)
Comment (W1ns-Smile) [reply src/GwentTracker/locale/cards.ru.po]: https://vedmak.fandom.com/wiki/Фледер

File: src/GwentTracker/locale/cards.ru.po (hunk 8 above)
Comment (Rfvgyhn) [reply src/GwentTracker/locale/cards.ru.po]: I'm not sure what you're trying to point out with the images. My comments just meant that these changes don't match what's in the game. Uppercase vs lowercase. `Р` and `р`, `К` and `к`, etc...

Comment (denis-ershov) [reply src/GwentTracker/locale/cards.ru.po]: I just did not immediately understand that we are talking about lower and upper case

Comment (rfvgyhn) [reply src/GwentTracker/locale/cards.ru.po]: Yeah, sorry about that. I didn't make it very clear. And for some reason the Eredin card uses the in-game elder script in Russian and Polish while a few of the languages use a translation of "I've long awaited this...", and others use a translation of "Get us out of here! Already!". No idea why.

Comment (denis-ershov) [reply src/GwentTracker/locale/cards.ru.po]: Question on the interface, what do you mean by Cutoff?

Comment (rfvgyhn) [reply src/GwentTracker/locale/cards.ru.po]: The point at which you can no longer obtain the card. So if you complete the quest _Reason of State_ you can no longer obtain the _Geralt of Rivia_ card.

File: src/GwentTracker/locale/cards.ru.po [hunk 9]
```diff
@@ -485,7 +485,7 @@ msgstr "Я могу стать твоей последней - и самой л
 
 #: src/GwentTracker/data/monsters.yml:488 -> Card.name
 msgid "Eredin King of the Wild Hunt"
-msgstr "Эредин Бреакк Глас Король Aen Elle"
+msgstr "Эредин Бреакк Глас Командир Дикой Охоты"
```
Comment (rfvgyhn) [inline src/GwentTracker/locale/cards.ru.po]: I think the Eredin card titles got mixed up with each other. Lines 488 with 500 and 512 with 520.

Comment (denis-ershov) [reply src/GwentTracker/locale/cards.ru.po]: Yes, sorry, I checked the names in the old card base.

**Pull request merged**